webpack/assets/javascripts/react_app/components/LoginPage/__tests__/__snapshots__/LoginPage.test.js.snap

```diff
@@ -0,0 +1,75 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AutoComplete rendering renders LoginPage 1`] = `
+<LoginPage
+  card={
+    Object {
+      "form": Object {
+        "additionalFields": <input
+          name="authenticity_token"
+          type="hidden"
+          value="1234"
+    />,
+        "attributes": Object {
+          "action": "/users/login",
+          "method": "post",
+        },
+        "disableSubmit": false,
+        "onSubmit": [Function],
+        "passwordField": Object {
+          "attributes": Object {
+            "name": "login[password]",
+          },
+          "errors": Object {},
+          "id": "login_password",
+          "placeholder": "Password",
+          "showWarning": false,
+          "type": "password",
+          "warning": null,
+          "warnings": Object {},
+        },
+        "showError": false,
+        "submitError": null,
+        "submitText": "Log In",
+        "usernameField": Object {
+          "attributes": Object {
+            "name": "login[login]",
+          },
+          "error": null,
+          "errors": Object {},
+          "id": "login_login",
+          "placeholder": "Username",
+          "showError": false,
+          "type": "text",
+        },
+        "validate": false,
+      },
+      "title": "Log into your account",
+    }
+  }
+  container={
+    Object {
+      "alert": Object {
+        "message": null,
+        "onDismiss": null,
+        "show": false,
+      },
+      "backgroundUrl": "/assets/login_background.jpg",
+    }
+  }
+  footerLinks={Array []}
+  header={
+    Object {
+      "caption": <Unknown>
```

Comment:
Unknown ?

Reply:
A `React.Fragment`,
see the inner elements below.

Reply:
Seems that it got fixed / readable with the new Jest version.
cc @ohadlevy